x-pack/plugins/security_solution/public/flyout/right/components/header_title.tsx

```diff
@@ -7,20 +7,45 @@
 
 import type { FC } from 'react';
 import React, { memo } from 'react';
-import { EuiSpacer, EuiTitle } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { isEmpty } from 'lodash';
+import { DocumentSeverity } from './severity';
+import { RiskScore } from './risk_score';
+import { DOCUMENT_DETAILS } from './translations';
+import { useBasicDataFromDetailsData } from '../../../timelines/components/side_panel/event_details/helpers';
+import { useRightPanelContext } from '../context';
+import { PreferenceFormattedDate } from '../../../common/components/formatted_date';
 import { FLYOUT_HEADER_TITLE_TEST_ID } from './test_ids';
-import { HEADER_TITLE } from '../translations';
 
 /**
  * Document details flyout right section header
  */
 export const HeaderTitle: FC = memo(() => {
+  const { dataFormattedForFieldBrowser } = useRightPanelContext();
+  const { isAlert, ruleName, timestamp } = useBasicDataFromDetailsData(
+    dataFormattedForFieldBrowser
+  );
+
+  if (!dataFormattedForFieldBrowser) {
+    return <></>;
```

Comment:
`null` would work just as well :)

Reply:
looking at some [docs](https://legacy.reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering), it seems that `null` is preferred over `React.Fragment`, thanks for pointing this out! 👍